client/packages/system/src/Report/api/hooks/utils/usePrintReport.ts

```diff
@@ -21,8 +26,11 @@ export const usePrintReport = () => {
       const url = `${Environment.FILE_URL}${fileId}`;
       const win = window.open(url, '_blank');
       if (win) {
-        win.focus();
-        // win.print(); // crashes chrome if the file is a PDF :shrug:
+        if (EnvUtils.platform === Platform.Android) {
+          win.print(); // crashes chrome if the file is a PDF :shrug:
```

Comment:
this didn't actually pop up the print dialog like it has on desktop but I've left it in. at least chrome isn't chrashing.